docs/reports/v0.4.0/categories.svg

```diff
@@ -0,0 +1,175 @@
+L 510 93" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="99" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">53</text><path  d="M 505 90
+L 510 90" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="96" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">54</text><path  d="M 505 88
+L 510 88" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="94" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">55</text><path  d="M 505 85
+L 510 85" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="91" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">56</text><path  d="M 505 83
+L 510 83" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="89" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">57</text><path  d="M 505 80
+L 510 80" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="86" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">58</text><path  d="M 505 78
+L 510 78" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="84" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">59</text><path  d="M 505 75
+L 510 75" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="81" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">60</text><path  d="M 505 73
+L 510 73" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="79" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">61</text><path  d="M 505 70
+L 510 70" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="76" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">62</text><path  d="M 505 68
+L 510 68" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="74" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">63</text><path  d="M 505 66
+L 510 66" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="72" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">64</text><path  d="M 20 225
```

Comment:
Should we delete some y axis ticks?

Reply:
You can do that with a follow up if you like.

Reply:
No. just delete the y axis and give the bars numbers. that is much better to read